baseframe/assets.py

```diff
@@ -300,6 +306,15 @@
     'select2-baseframe.js',
     'getdevicepixelratio.js'
 ]}
+assets['extra-material.js'][Version('0.0.0')] = {'requires': [
+    'jquery-modal.js',
+    'jquery.form.js',
+    'jquery.autosize.js',
+    'jquery.timepicker.js',
+    'select2-material.js',
+    'jquery.tinymce.js>=3.0.0,<4.0.0',
```

Comment:
TinyMCE should not be bundled with extras unless the app explicitly wants it. TinyMCE3 is also obsolete and only present because of unmaintained sites like [Kharcha](https://expenses.hasgeek.com).

Reply:
Fixed